### ISSUE TITLE
Videos won't load.

### ISSUE THREAD

**Issue by zappathustra** [luakit/luakit#814]
Most of the time, when I want to watch a video on YouTube (but it happens elsewhere), the first few seconds are loaded but nothing more (and I switch to Firefox). Sometimes the video works properly, but that's rather unfrequent.

I've had this bug for a long time but I'm only reporting it now. Doesn't happen on Epiphany.

**Comment by aidanholm** [comment]
What version of webkit is this on?

**Comment by zappathustra** [comment]
2.24 (still won't do to move to 2.26 on Arch).

**Comment by aidanholm** [comment]
That's quite peculiar; I've never had any issues like that. Unfortunately I'm not sure what to suggest other than trying a default config/settings. With youtube, I'd also tend to suspect user agent detection, so you could try changing your UA and seeing if that fixes things.

**Comment by zappathustra** [comment]
My `settings.webview.user_agent` is an empty string, so it's set to the default WebKit UA. Setting it to another one, like Firefox, doesn't change anything. Nor does starting with an empty `userconf.lua`

Also, reloading the page sometimes works, but it's not the most frequent case.

So we're stuck. I'll post what I find here on the subject, if I find anything.

**Comment by c0dev0id** [comment]
I had similiar issues. Is there a chance you're using intel integrated graphics?
Try setting `export LIBGL_DRI3_DISABLE=1` before starting luakit. It fixed these issues for me.

**Comment by zappathustra** [comment]
Actually the problem has almost disappeared, on YouTube at least. Videos elsewhere can be buggy. I'll try you fix, thanks.

**Comment by c0dev0id** [comment]
The issue seems to not occur anymore for the author. I'm closing this issue. Feel free to reopen if there are further questions.
